src/components/split-button/split-button-test.stories.tsx

```diff
@@ -53,44 +54,39 @@ export default {
   },
 };
 
-/* eslint-disable react/prop-types */
+type SplitButtonStoryArgs = {
+  buttonType?: SplitButtonProps["buttonType"];
+  textContent: string;
```

Comment:
I'm not sure what `textContent` is doing here - it's not used in the `SplitButtonStory` below (or any of the other stories), nor is it an accepted prop on `SplitButton`

Reply:
Hmm, I'm not sure. I just pulled this code out of the `split-button.test.js` file and into the `split-button-test.stories.tsx` file. I'll investigate.

Reply:
ah thanks, I hadn't noticed that 🤦 Worth investigating but probably not worth spending too long on, if we had this before I won't let it stop me approving!